README.md

```diff
@@ -20,7 +20,7 @@ Create you own `.env` file based on `.env.example`. You can fill it with your ow
 Launch the postgres DB used by the backend:
 
 ```sh
-docker compose up -d
+docker compose up -d db
```

Comment:
This would not launch the second container, Yente, which might be what we want for a speedy startup, but that will be missing a dependency for Sanctions Check.

Yente was added after the fact, and I think it was the introduction that was wrong.

Reply:
done